File: examples/verify-claim.ts
import { VeroqClient } from '@veroq/sdk';

const client = new VeroqClient();
const result = await client.verify("NVIDIA beat Q4 earnings expectations");

console.log(`Verdict: ${result.verdict} (${Math.round(result.confidence * 100)}%)`);
for (const e of result.evidence_chain || []) {
  console.log(`  [${e.position}] ${e.source}: "${e.snippet.slice(0, 80)}..."`);
}

File: examples/verify-claim.py
"""Verify any claim with evidence chain and confidence breakdown."""
from veroq import VeroqClient

client = VeroqClient()
result = client.verify("The Federal Reserve held rates steady in March 2026")

print(f"Verdict: {result['verdict']} ({result['confidence']*100:.0f}%)")
print(f"\nConfidence breakdown:")
for k, v in result["confidence_breakdown"].items():
    print(f"  {k}: {v*100:.0f}%")
print(f"\nEvidence chain ({len(result['evidence_chain'])} sources):")
for e in result["evidence_chain"]:
    print(f"  [{e['position']}] {e['source']}: \"{e['snippet'][:80]}...\"")
